soap/src/java/com/zimbra/soap/admin/message/GetAccountRequest.java

```diff
@@ -77,6 +88,14 @@ public GetAccountRequest(AccountSelector account, Boolean applyCos,
         this.applyCos = ZmBoolean.fromBool(applyCos);
```

Comment:
change to call new `GetAccountRequest` defined at line 91

Reply:
Done

Reply:
Lines 84-96:
move statements from GetAccountRequest defined at line 84 to another at line 91, and call the new GetAccountRequest with `effectiveQuota false`.
```
    public GetAccountRequest(AccountSelector account, Boolean applyCos,
            Iterable<String> attrs) {
        this(account, applyCos, false, (Iterable<String>) null);
    }

    public GetAccountRequest(AccountSelector account, Boolean applyCos,
            Boolean effectiveQuota, Iterable<String> attrs) {
        super(attrs);
        this.account = account;
        this.applyCos = ZmBoolean.fromBool(applyCos);
        this.effectiveQuota = ZmBoolean.fromBool(effectiveQuota);
    }
```

Reply:
Done